shipyard_blueprints/postgresql/shipyard_postgresql/cli/execute_sql.py

```diff
@@ -39,53 +32,34 @@ def get_args():
     return args
 
 
-def create_connection_string(args):
-    """
-    Set the database connection string as an environment variable using the keyword arguments provided.
-    This will override system defaults.
-    """
-    if args.db_connection_url:
-        os.environ["DB_CONNECTION_URL"] = args.db_connection_url
-    elif args.host and args.database:
-        os.environ["DB_CONNECTION_URL"] = (
-            f"postgresql://{args.username}:{args.password}@{args.host}:{args.port}/{args.database}?{args.url_parameters}"
-        )
-
-    db_string = os.environ.get("DB_CONNECTION_URL")
-    return db_string
-
-
-def create_db_connection(db_string):
-    if "db.bit.io" in db_string:
-        db_connection = create_engine(
-            db_string, connect_args={"sslmode": "require"}, isolation_level="AUTOCOMMIT"
-        )
-    else:
-        db_connection = create_engine(db_string)
-    Session = sessionmaker(bind=db_connection)
-    return db_connection, Session
-
-
 def main():
-    args = get_args()
-    query = text(args.query)
-
-    db_string = create_connection_string(args)
     try:
-        db_connection, Session = create_db_connection(db_string)
-    except Exception as e:
-        print(f"Failed to connect to database {args.database}")
-        raise (e)
+        args = get_args()
+        query = text(args.query)
+        client_args = {
+            "user": args.username,
+            "pwd": args.password,
+            "host": args.host,
+            "database": args.database,
+            "port": args.port,
+            "url_params": args.url_parameters if args.url_parameters != "" else None,
+        }
+        postgres = PostgresClient(**client_args)
+        postgres.execute_query(query)
+        logger.info("Successfully executed query")
+
+    except ExitCodeException as ec:
+        logger.error(ec.message)
+        sys.exit(ec.exit_code)
 
-    session = Session()
-    try:
-        session.execute(query)
-        session.commit()
-        print("Your query has been successfully executed.")
     except Exception as e:
-        print(f"Failed to execute query {args.query} due to {e}")
+        logger.error(
+            f"An unexpected error occurred when attempting to execute query in Postgres. Message from the server reads: {e}"
+        )
+        sys.exit(Database.EXIT_CODE_UNKNOWN)
+
     finally:
-        session.close()
+        postgres.close()
```

Comment:
`postgres` may have to be initialized so that you can do a `if postrgres: postrgres.close` otherise this could throw a confusing traceback error for the customer